performance_manager/lib/postgres_utils.py

```diff
@@ -147,13 +210,24 @@ def __init__(
             # this is required for foreign key support in sqlite, per this:
             # https://docs.sqlalchemy.org/en/14/dialects/sqlite.html#foreign-key-support
             @sa.event.listens_for(sa.engine.Engine, "connect")
-            def set_sqlite_pragma(dbapi_connection, _):  # type: ignore
-                cursor = dbapi_connection.cursor()
-                cursor.execute("PRAGMA foreign_keys=ON")
-                cursor.close()
+            def set_sqlite_pragma(
+                connection: sa.engine.Connection, _: Any
+            ) -> None:
+                connection.execute("PRAGMA foreign_keys=ON")
+
+            sa.event.listen(
+                sa.schema.Table,
+                "after_create",
+                sqlite_create_modified_trigger,
+            )
```

Comment:
sqlalchemy event that calls trigger creation function, will be called after any table creation event, passing in the table that was created as an argument

Reply:
if we're connecting to a table thats already been created, does that mean we won't be applying this function?

Reply:
Correct, it's only called during table creation

Reply:
cool. we'll probably need to create a util that can drop the tables off of the RDS so that we can reinstate them from the ECS when we merge this.

Reply:
I think we should look into using something like alembic https://pypi.org/project/alembic/ to handle schema migrations. I originally assumed this type of functionality was built into sqlalchemy, but it looks like they developed another tool instead.